packages/core/admin/admin/src/components/AuthenticatedApp/index.js

```diff
@@ -15,7 +15,7 @@ import {
 } from './utils/api';
 import checkLatestStrapiVersion from './utils/checkLatestStrapiVersion';
 import { getFullName } from '../../utils';
-import { handleGuidedTourVisibility } from '../GuidedTour/utils/handleGuidedTourVisibility';
+import handleGuidedTourVisibility from '../GuidedTour/utils/handleGuidedTourVisibility';
```

Comment:
I think that you can move the logic in the AuthenticatedApp directly by removing the utility function, IMO it creates noise.

Can you move the logic in the effect directly?

```js
useEffect(() => {
  if (userRoles) {
    const isUserSuperAdmin = userRoles.find(({ code }) => code === "strapi-super-admin" );
        
        if (isUserSuperAdmin) {
            setGuidedTourVisibilityRef.current(true);
       }
        }     
 
 
  }
},[]);
```